tensorflow/lite/interpreter_builder.cc

```diff
@@ -564,11 +564,13 @@ TfLiteStatus InterpreterBuilder::ParseTensors(
       // TODO(aselle): Check what happens if we have an unspecified size
       // constant.
       *buffer_data = nullptr;
-      if (tensor->buffer() == 0) return kTfLiteOk;
-      if (tensor->buffer() >= buffers->size()) {
+      if (tensor->buffer() == 0) {
+        return kTfLiteOk;
+      }
+      if (!buffers || tensor->buffer() >= buffers->size()) {
```

Comment:
I show the detail in the comment.
https://github.com/tensorflow/tensorflow/pull/47754/commits/582bf0d3ac33fc10156f737c0d42f3adee54409a

If we build with cmake, we can't generate the mutable schema header for the original UpdateOpVersion() call. So, we can't call the UpdateOpVersion() in cmake build.

If I change the UpdateOpVersion() implementation using the pack/unpack with immutable schema header, I need the additional --force-empty-vectors flag for the immutable header. Otherwise, some vector field will become null-ptr if the vector size is 0 for serialization.
I don't regenerate the immutable schema header here, so I need to relax the buffers checking in interpreter_builder.cc. Some kernel tests doesn't have buffer. The buffer field will become null-ptr if the model comes from my new UpdateOpVersion() pack/unpack implementation.